python/mdvtools/conversions.py

```
@@ -424,16 +533,20 @@
     )
     return p
 
-
-def _create_dt_heatmap(folder, project, mdv, marks):
+def _create_dt_heatmap(
+        project,
+        matrix,
+        ds="elements",
+        ifields=["chromosome", "start", "end"]
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)